lookout/style/typos/benchmarks/typo_commits_report.py

```diff
@@ -0,0 +1,193 @@
+"""Facilities to report the quality of a given model on a given dataset."""
+import csv
+import json
+import logging
+import os
+from typing import Any, Dict, Iterable, Iterator, List, Optional, Sequence, Tuple
+
+
+from lookout.core.analyzer import ReferencePointer
+from lookout.core.api.service_analyzer_pb2 import Comment
+from lookout.core.api.service_data_pb2 import Change, File
+from lookout.core.data_requests import DataService, request_files
+import pandas
+
+from lookout.style.cloner import Cloner
+from lookout.style.format.benchmarks.quality_report import handle_input_arg
+from lookout.style.format.utils import generate_comment
+from lookout.style.reporter import Reporter
+from lookout.style.typos import IdTyposAnalyzer
+from lookout.style.typos.analyzer import TypoFix
+
+
+class TyposAnalyzerSpy(IdTyposAnalyzer):
+    """
+    The Analyzer which returns fixes found by IdTyposAnalyzer for all file content as JSON \
```

Comment:
```suggestion
    The Analyzer which returns fixes found by IdTyposAnalyzer as JSON \
```

Reply:
it is an important part because we are looking for a fix in the diff only by default. I think about how to make it clearer.